### ISSUE TITLE
4.0.8 not recording on 11.1 and 11.2

### ISSUE THREAD

**Issue by omarpiani** [wakatime/netbeans-wakatime#16]
I'm not sure but I think was working in 11.0

```
-------------------------------------------------------------------------------
>Log Session: Monday, November 11, 2019 12:00:56 PM CET
>System Info: 
  Product Version         = Apache NetBeans IDE 11.2
  Operating System        = Windows 10 version 10.0 running on amd64
  Java; VM; Vendor        = 1.8.0_171; Java HotSpot(TM) 64-Bit Server VM 25.171-b11; Oracle Corporation
  Runtime                 = Java(TM) SE Runtime Environment 1.8.0_171-b11
  Java Home               = C:\Program Files\Java\jdk1.8.0_171\jre
  System Locale; Encoding = it_IT (nb); Cp1252
  Home Directory          = C:\Users\omar.piani
  Current Directory       = C:\Program Files\NetBeans-11.2
  User Directory          = C:\Users\omar.piani\AppData\Roaming\NetBeans\11.2
  Cache Directory         = C:\Users\omar.piani\AppData\Local\NetBeans\Cache\11.2
  Installation            = C:\Program Files\NetBeans-11.2\netbeans\nb
                            C:\Program Files\NetBeans-11.2\netbeans\ide
                            C:\Program Files\NetBeans-11.2\netbeans\extide
                            C:\Program Files\NetBeans-11.2\netbeans\webcommon
                            C:\Program Files\NetBeans-11.2\netbeans\websvccommon
                            C:\Program Files\NetBeans-11.2\netbeans\php
                            C:\Program Files\NetBeans-11.2\netbeans\harness
                            C:\Program Files\NetBeans-11.2\netbeans\platform
  Boot & Ext. Classpath   = C:\Program Files\Java\jdk1.8.0_171\jre\lib\resources.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\rt.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\sunrsasign.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\jsse.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\jce.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\charsets.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\jfr.jar;C:\Program Files\Java\jdk1.8.0_171\jre\classes;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\access-bridge-64.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\cldrdata.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\dnsns.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\jaccess.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\jfxrt.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\localedata.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\nashorn.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\sunec.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\sunjce_provider.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\sunmscapi.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\sunpkcs11.jar;C:\Program Files\Java\jdk1.8.0_171\jre\lib\ext\zipfs.jar
  Application Classpath   = C:\Program Files\NetBeans-11.2\netbeans\platform\lib\boot.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\lib\org-openide-modules.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\lib\org-openide-util-lookup.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\lib\org-openide-util-ui.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\lib\org-openide-util.jar;C:\Program Files\Java\jdk1.8.0_171\lib\dt.jar;C:\Program Files\Java\jdk1.8.0_171\lib\tools.jar
  Startup Classpath       = C:\Program Files\NetBeans-11.2\netbeans\platform\core\asm-all-5.0.1.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\core\core-base.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\core\core.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\core\org-netbeans-libs-asm.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\core\org-openide-filesystems-compat8.jar;C:\Program Files\NetBeans-11.2\netbeans\platform\core\org-openide-filesystems.jar;C:\Program Files\NetBeans-11.2\netbeans\nb\core\org-netbeans-upgrader.jar;C:\Program Files\NetBeans-11.2\netbeans\nb\core\locale\core_nb.jar
-------------------------------------------------------------------------------
INFO [org.netbeans.modules.netbinox]: Install area set to file:/C:/Program Files/NetBeans-11.2/netbeans/
WARNING [org.netbeans.core.modules]: the modules [org.netbeans.modules.xml.text] use org.netbeans.modules.editor.deprecated.pre65formatting which is deprecated.
WARNING [org.netbeans.core.modules]: the modules [org.netbeans.modules.ide.kit, org.netbeans.modules.xml.text] use org.netbeans.modules.editor.structure which is deprecated.
WARNING [org.openide.filesystems.Ordering]: Found same position 127 for both Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageLockJsonDataObject-Registration.xml and Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-NpmDebugLogDataObject-Registration.xml
WARNING [org.openide.filesystems.Ordering]: Found same position 127 for both Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageLockJsonDataObject-Registration.xml and Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageJsonDataObject-Registration.xml
WARNING [org.openide.filesystems.Ordering]: Found same position 127 for both Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-NpmDebugLogDataObject-Registration.xml and Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageJsonDataObject-Registration.xml
WARNING [null]: Last record repeated again.
WARNING [org.openide.filesystems.Ordering]: Found same position 127 for both Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-NpmDebugLogDataObject-Registration.xml and Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageLockJsonDataObject-Registration.xml
WARNING [org.openide.filesystems.Ordering]: Found same position 127 for both Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageJsonDataObject-Registration.xml and Services/MIMEResolver/org-netbeans-modules-javascript-nodejs-file-PackageLockJsonDataObject-Registration.xml
INFO [org.netbeans.core.startup.NbEvents]: Turning on modules:
	org.openide.util.lookup [8.39 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.openide.util [9.13 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.openide.util.ui [9.14 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.openide.modules [7.54 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.api.progress/1 [1.53 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]

[...]

	org.netbeans.modules.project.indexingbridge [1.23 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.spi.tasklist/1 [1.41.0.1 1 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.modules.parsing.indexing [9.14.0.1.3.8 1 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.wakatime.netbeans.plugin [4.0.8 201605120406 160512]
	org.openide.execution [9.9 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.openide.execution.compat8 [9.8 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.upgrader [4.41 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.swing.dirchooser [1.35 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.spi.viewmodel/2 [1.57 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.spi.palette/1 [1.52 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.spi.navigator/1 [1.44 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.modules.editor.settings.lib [1.57.0.1 1 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	org.netbeans.modules.editor.fold/1 [1.51 11.2-d04fb24027334c4b6fd8397b5d0cdd33187a8f54]
	
[...]

INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy: fell to default (correct if direct mode went before)
INFO [org.openide.loaders.FolderInstance.Services]
java.lang.InterruptedException
	at java.lang.Object.wait(Native Method)
	at org.openide.util.Task.waitFinished(Task.java:139)
	at org.openide.util.RequestProcessor$Task.waitFinished(RequestProcessor.java:1711)
	at org.openide.util.Task.waitFinished(Task.java:179)
[catch] at org.openide.loaders.FolderLookup$ProxyLkp.beforeLookup(FolderLookup.java:403)
	at org.openide.util.lookup.ProxyLookup.beforeLookup(ProxyLookup.java:184)
	at org.openide.util.lookup.ProxyLookup$R.myBeforeLookup(ProxyLookup.java:650)
	at org.openide.util.lookup.ProxyLookup$R.computeResult(ProxyLookup.java:528)
	at org.openide.util.lookup.ProxyLookup$R.allInstances(ProxyLookup.java:488)
	at org.openide.util.lookup.ProxyLookup$R.allInstances(ProxyLookup.java:484)
	at org.openide.util.lookup.ProxyLookup$LazyCollection.computeSingleResult(ProxyLookup.java:1264)
	at org.openide.util.lookup.ProxyLookup$LazyCollection.computeDelegate(ProxyLookup.java:1102)
	at org.openide.util.lookup.ProxyLookup$LazyCollection.delegate(ProxyLookup.java:1069)
	at org.openide.util.lookup.ProxyLookup$LazyCollection.delegate(ProxyLookup.java:1052)
	at org.openide.util.lookup.ProxyLookup$LazyCollection.size(ProxyLookup.java:1137)
	at org.netbeans.ProxyURLStreamHandlerFactory.resultChanged(ProxyURLStreamHandlerFactory.java:149)
	at org.netbeans.ProxyURLStreamHandlerFactory.createURLStreamHandler(ProxyURLStreamHandlerFactory.java:131)
	at org.eclipse.osgi.framework.internal.protocol.StreamHandlerFactory.createURLStreamHandler(StreamHandlerFactory.java:122)
	at java.net.URL.getURLStreamHandler(URL.java:1142)
	at java.net.URL.<init>(URL.java:599)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at java.net.URI.toURL(URI.java:1089)
	at org.openide.loaders.DataShadow.findOriginal(DataShadow.java:375)
	at org.openide.loaders.DataShadow.deserialize(DataShadow.java:441)
	at org.openide.loaders.DataLoaderPool$ShadowLoader.createMultiObject(DataLoaderPool.java:1029)
	at org.openide.loaders.MultiFileLoader.handleFindDataObject(MultiFileLoader.java:128)
	at org.openide.loaders.DataObjectPool.handleFindDataObject(DataObjectPool.java:136)
	at org.openide.loaders.DataLoader.findDataObject(DataLoader.java:382)
	at org.openide.loaders.DataLoader.findDataObject(DataLoader.java:355)
	at org.openide.loaders.DataLoaderPool.findDataObject(DataLoaderPool.java:510)
	at org.openide.loaders.DataLoaderPool.findDataObject(DataLoaderPool.java:468)
	at org.openide.loaders.DataObject.find(DataObject.java:601)
	at org.netbeans.core.NbKeymap.idForFile(NbKeymap.java:419)
	at org.netbeans.core.NbKeymap.bindings(NbKeymap.java:238)
	at org.netbeans.core.NbKeymap.keyStrokeForAction(NbKeymap.java:380)
	at org.netbeans.core.NbKeymap$AcceleratorBindingImpl.keyStrokeForAction(NbKeymap.java:396)
	at org.openide.awt.AcceleratorBinding.setAccelerator(AcceleratorBinding.java:65)
	at org.openide.awt.MenuBar$MenuBarFolder.convertToComponent(MenuBar.java:464)
	at org.openide.awt.MenuBar$MenuBarFolder.createInstance(MenuBar.java:440)
	at org.openide.loaders.FolderInstance.defaultProcessObjectsFinal(FolderInstance.java:863)
	at org.openide.loaders.FolderInstance$1R.run(FolderInstance.java:705)
	at org.openide.util.Task.run(Task.java:232)
	at org.netbeans.modules.openide.loaders.AWTTask.run(AWTTask.java:54)
	at org.netbeans.modules.openide.loaders.AWTTask.waitFinished(AWTTask.java:93)
	at org.openide.loaders.FolderInstance$1R.waitFinished(FolderInstance.java:731)
	at org.netbeans.modules.openide.loaders.AWTTask.waitFor(AWTTask.java:118)
	at org.openide.loaders.FolderInstance.waitFinished(FolderInstance.java:315)
	at org.openide.awt.MenuBar.getMenuCount(MenuBar.java:182)
	at javax.swing.plaf.basic.BasicMenuBarUI.installListeners(BasicMenuBarUI.java:97)
	at com.sun.java.swing.plaf.windows.WindowsMenuBarUI.installListeners(WindowsMenuBarUI.java:120)
	at javax.swing.plaf.basic.BasicMenuBarUI.installUI(BasicMenuBarUI.java:74)
	at javax.swing.JComponent.setUI(JComponent.java:666)
	at javax.swing.JMenuBar.setUI(JMenuBar.java:144)
	at javax.swing.JMenuBar.updateUI(JMenuBar.java:153)
	at org.openide.awt.MenuBar.updateUI(MenuBar.java:170)
	at org.openide.awt.MenuBar$1.run(MenuBar.java:150)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:136)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
INFO [WakaTime]: Initializing WakaTime plugin v4.0.8 (https://wakatime.com/)
CONFIG [WakaTime]: Logging level set to DEBUG
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\omar.piani\AppData\Roaming\WakaTime\wakatime-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: WakaTime Core Location: C:\Users\omar.piani\AppData\Roaming\WakaTime\wakatime-master\wakatime\cli.py
CONFIG [WakaTime]: WakaTime Core Exists: true
CONFIG [WakaTime]: Found python binary: pythonw
CONFIG [WakaTime]: wakatime cli version check output: "13.0.1"
CONFIG [WakaTime]: wakatime cli version check exit code: 0
CONFIG [WakaTime]: Current cli version from GitHub: 13.0.1
INFO [WakaTime]: wakatime-cli is up to date.
CONFIG [WakaTime]: Python location: pythonw
CONFIG [WakaTime]: CLI location: C:\Users\omar.piani\AppData\Roaming\WakaTime\wakatime-master\wakatime\cli.py
CONFIG [WakaTime]: API Key: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX62a6
INFO [WakaTime]: Finished initializing WakaTime plugin.
INFO [WakaTime]: Checking for updates to WakaTime plugin...
WARNING [org.netbeans.modules.progress.spi.InternalHandle]: Cannot call progress on a task that was never started at org.netbeans.modules.autoupdate.services.UpdateUnitProviderImpl.refresh(UpdateUnitProviderImpl.java:152)
WARNING [null]: Last record repeated again.
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
INFO [WakaTime]: WakaTime plugin is up to date.
Diagnostic information
Input arguments:
	-Dnetbeans.importclass=org.netbeans.upgrade.AutoUpgrade
	-XX:+UseStringDeduplication
	-Djdk.lang.Process.allowAmbiguousCommands=true
	-Xss2m
	-Djdk.gtk.version=2.2
	-Dapple.laf.useScreenMenuBar=true
	-Dapple.awt.graphics.UseQuartz=true
	-Dsun.java2d.noddraw=true
	-Dsun.java2d.dpiaware=true
	-Dsun.zip.disableMemoryMapping=true
	-Dplugin.manager.check.updates=false
	-Dnetbeans.extbrowser.manual_chrome_plugin_install=yes
	--add-opens=java.base/java.net=ALL-UNNAMED
	--add-opens=java.base/java.lang.ref=ALL-UNNAMED
	--add-opens=java.base/java.lang=ALL-UNNAMED
	--add-opens=java.base/java.security=ALL-UNNAMED
	--add-opens=java.base/java.util=ALL-UNNAMED
	--add-opens=java.desktop/javax.swing.plaf.basic=ALL-UNNAMED
	--add-opens=java.desktop/javax.swing.text=ALL-UNNAMED
	--add-opens=java.desktop/javax.swing=ALL-UNNAMED
	--add-opens=java.desktop/java.awt=ALL-UNNAMED
	--add-opens=java.desktop/java.awt.event=ALL-UNNAMED
	--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED
	--add-opens=jdk.jshell/jdk.jshell=ALL-UNNAMED
	--add-modules=jdk.jshell
	--add-exports=java.desktop/sun.awt=ALL-UNNAMED
	--add-exports=java.desktop/java.awt.peer=ALL-UNNAMED
	--add-exports=java.desktop/com.sun.beans.editors=ALL-UNNAMED
	--add-exports=java.desktop/sun.swing=ALL-UNNAMED
	--add-exports=java.desktop/sun.awt.im=ALL-UNNAMED
	--add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED
	--add-exports=java.management/sun.management=ALL-UNNAMED
	--add-exports=java.base/sun.reflect.annotation=ALL-UNNAMED
	--add-exports=jdk.javadoc/com.sun.tools.javadoc.main=ALL-UNNAMED
	-XX:+IgnoreUnrecognizedVMOptions
	-Djdk.home=C:\Program Files\Java\jdk1.8.0_171
	-Dnetbeans.home=C:\Program Files\NetBeans-11.2\netbeans\platform
	-Dnetbeans.user=C:\Users\omar.piani\AppData\Roaming\NetBeans\11.2
	-Dnetbeans.default_userdir_root=C:\Users\omar.piani\AppData\Roaming\NetBeans
	-XX:+HeapDumpOnOutOfMemoryError
	-XX:HeapDumpPath=C:\Users\omar.piani\AppData\Roaming\NetBeans\11.2\var\log\heapdump.hprof
	-Dsun.awt.keepWorkingSetOnMinimize=true
	-Dnetbeans.dirs=C:\Program Files\NetBeans-11.2\netbeans\nb;C:\Program Files\NetBeans-11.2\netbeans\ergonomics;C:\Program Files\NetBeans-11.2\netbeans\ide;C:\Program Files\NetBeans-11.2\netbeans\extide;C:\Program Files\NetBeans-11.2\netbeans\java;C:\Program Files\NetBeans-11.2\netbeans\apisupport;C:\Program Files\NetBeans-11.2\netbeans\webcommon;C:\Program Files\NetBeans-11.2\netbeans\websvccommon;C:\Program Files\NetBeans-11.2\netbeans\enterprise;C:\Program Files\NetBeans-11.2\netbeans\mobility;C:\Program Files\NetBeans-11.2\netbeans\profiler;C:\Program Files\NetBeans-11.2\netbeans\python;C:\Program Files\NetBeans-11.2\netbeans\php;C:\Program Files\NetBeans-11.2\netbeans\identity;C:\Program Files\NetBeans-11.2\netbeans\harness;C:\Program Files\NetBeans-11.2\netbeans\cnd;C:\Program Files\NetBeans-11.2\netbeans\cndext;C:\Program Files\NetBeans-11.2\netbeans\dlight;C:\Program Files\NetBeans-11.2\netbeans\groovy;C:\Program Files\NetBeans-11.2\netbeans\extra;C:\Program Files\NetBeans-11.2\netbeans\javacard;C:\Program Files\NetBeans-11.2\netbeans\javafx
	exit
Compiler: HotSpot 64-Bit Tiered Compilers
Heap memory usage: initial 254,0MB maximum 3609,0MB
Non heap memory usage: initial 2,4MB maximum -1b
Garbage collector: PS Scavenge (Collections=8 Total time spent=0s)
Garbage collector: PS MarkSweep (Collections=3 Total time spent=0s)
Classes: loaded=8852 total loaded=8852 unloaded 0
INFO [org.netbeans.core.ui.warmup.DiagnosticTask]: Total memory 17.024.614.400
INFO [org.netbeans.modules.php.editor.parser.GSFPHPParser]: Parsing of big PHP files enabled: false (max size: 5.000.000)
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Resolving dependencies took: 20 ms
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Complete indexing of 0 binary roots took: 0 ms
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: jar:file:/C:/Program%20Files/NetBeans-11.2/netbeans/php/docs/phpsigfiles.zip!/phpstubs/phpruntime/ took: 194 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: nbfs://nbhost/SystemFileSystem/PHP/RuntimeLibraries/ took: 20 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: C:\Users\omar.piani\source\repos\Proval took: 561 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 2 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: C:\Users\omar.piani\source\repos\Proval\tests took: 30 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: C:\Users\omar.piani\source\repos\easybadge took: 703 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: C:\Users\omar.piani\source\repos\easybadge\test took: 35 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Indexing of: C:\Users\omar.piani\source\repos\gustait took: 1.585 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 0 ms]
INFO [org.netbeans.modules.parsing.impl.indexing.RepositoryUpdater]: Complete indexing of 7 source roots took: 3.128 ms (New or modified files: 0, Deleted files: 0) [Adding listeners took: 2 ms]
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
WARNING [org.openide.filesystems.Ordering]: Not all children in OptionsDialog/ marked with the position attribute: [org-wakatime-netbeans-plugin-WakaTimeOptionsPanelController.instance], but some are: [Actions, Advanced, Appearance, Editor, Html5, Java, Keywords, PreviewExamples, Team, org-netbeans-modules-php-project-ui-options-PHPOptionsCategory, General.instance, Editor.instance, FontsAndColors.instance, Keymaps.instance, Java.instance, Html5.instance, org-netbeans-modules-php-project-ui-options-PHPOptionsCategory.instance, Team.instance, Appearance.instance, Advanced.instance]
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: L'elemento radice "module_updates" del documento deve corrispondere alla radice DOCTYPE "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Documento non valido: nessuna grammatica trovata.
```

**Comment by omarpiani** [comment]
~/.wakatime.log have rows only regarding visualstudio

**Comment by alanhamlett** [comment]
Fixed in v4.0.9.
